keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -163,29 +169,50 @@ type ControllerLeaderConfiguration struct {
 	// E.g. service-controller, route-controller, cloud-node-controller, etc
 	Name string `json:"name"`
 
-	// Component is the name of the component in which the controller should be running.
+	// Component is the name of the component in which the controller will be running.
 	// E.g. kube-controller-manager, cloud-controller-manager, etc
 	Component string `json:"component"`
 }
 ```
+#### Default LeaderMigrationConfiguration
+
+The `controller-manager` will provide `kube-controller-manager` and `cloud-controller-manager`
+each a default `LeaderMigrationConfiguration` that represents the situation where the controller manager is running with
+default assignments of controllers and lock type selection.
+
+The default values must be only used when no configuration file is specified. If a custom configuration file is
+specified to either controller manager, the specified configuration will completely replace default value for the
+corresponding controller manager.
 
 #### Component Flags
 
-The LeaderMigrationConfiguration type will be read by the `kube-controller-manager` and the `cloud-controller-manager` via a new flag `--cloud-migration-config` which
-accepts a path to a file containing the LeaderMigrationConfiguration type in yaml.
+Both `kube-controller-manager` and `cloud-controller-manager` will add support for the following two flags for Leader
```

Comment:
"will add" -> "will get"

I don't believe either KCM or CCM have to add the support. The framework gives them the support.

Reply:
That's a good point. Thanks